app/config/FrontendAppConfig.scala

```diff
@@ -61,4 +61,6 @@ class FrontendAppConfig @Inject()(configuration: Configuration) {
   lazy val whitelistDestination: String = configuration.get[String]("filters.whitelist.destination")
   lazy val whitelistExcluded: String = configuration.get[String]("filters.whitelist.excluded")
```

Comment:
we are not allowed to use `whitelist` anymore due to Black History month - these need to be changed Whitelist --> Allowlist and Blacklist --> Denylist

Reply:
removed whitelisting altogether - as we don't need it...